### DISCUSSION TITLE
Plane stress for finite strain in tensor mechanics module

### DISCUSSION THREAD

**Discussion by tqcthai** [idaholab/moose#27340] [Q&A General]
Dear all,

I try to apply plane stress condition for finite strain problem in the Tensor mechanics module but didn't get success. Below is part of the code I used to specify it.

```
[GlobalParams]
  displacements = 'disp_x disp_y'
  out_of_plane_strain = strain_zz
[]

[Modules]
  [TensorMechanics/Master/mech]
    strain = FINITE
    add_variables = true
    new_system = true
    incremental = true
    formulation = TOTAL
    planar_formulation = WEAK_PLANE_STRESS
    save_in = 'resid_x resid_y'
  []
[]

[Variables]
  [strain_zz]
  []
[]

[AuxVariables]
  [resid_x]
  []
  [resid_y]
  []
  [nl_strain_zz]
    order = CONSTANT
    family = MONOMIAL
  []
[]

[AuxKernels]
 [strain_zz]
   type = RankTwoAux
   rank_two_tensor = total_strain
   variable = nl_strain_zz
   index_i = 2
   index_j = 2
  []
[]
```

The error message is: attempted to set unregistered parameter(s) for WeakPlaneStress object:  large_kinematics, stabilize_strain
Any idea to fix it?

Thanks.
QTT

**Comment by GiudGiud** [comment]
Hello

I dont think the new lagrangian formulation is supported with any plane formulation.

I ll fix the error message, you are not hitting the right one

Guillaume

**Reply by GiudGiud** [reply]
so to fix it, set new_system = false

**Reply by GiudGiud** [reply]
actually it should since this change
https://github.com/idaholab/moose/pull/21770

@hugary1995 thoughts?

**Reply by hugary1995** [reply]
We have the capability now, but I haven't ported it into the action/physics yet... Create an issue and assign me plz.

**Reply by GiudGiud** [reply]
on it.
